### DISCUSSION TITLE
Sub Tasks,

### DISCUSSION THREAD

**Discussion by pembertonc** [community/community#13127] [Projects and Issues]
The Include sub Tasks,  to allow decomposition to tasks to accomplish the main task.  
Eg,  Main Item Create Registration
      Sub1 Verify user name not already in use.
      Sub2 Verify User name meets minimal length ..
...
...

Something like this.   Nothing fancy just to allow grouping of task under a heading.

**Comment by nexushoratio** [comment]
Are you asking for something different from the nested tasks?   I'm new to GH in general, and only started using tasks within issues recently.  I'm  spending some time today reading through old discussions about tasks, hence commenting on this old post.  But I also can't tell if some of these are about tasks in issues (which I use) or tasks in projects (and I don't use projects).

- [ ] Top level task
  - [ ] Sub task 1
  - [ ] Sub task 2
- [ ] Another top level task
